tests/metagpt/test_incremental_dev.py

```diff
@@ -0,0 +1,372 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+@Time    : 2024/01/03
+@Author  : mannaandpoem
+@File    : test_incremental_dev.py
+"""
+import os
+import subprocess
+
+import pytest
+from typer.testing import CliRunner
+
+from metagpt.const import DATA_PATH
+from metagpt.logs import logger
+from metagpt.startup import app
+
+runner = CliRunner()
+
+
+def test_refined_simple_calculator():
+    project_path = f"{DATA_PATH}/simple_add_calculator"
+    check_or_create_base_tag(project_path)
+
+    args = [
```

Comment:
How many type of increment idea support? like add new feature/ change existed feature and so on. Can we list these and add them in the unittests?

And seems that the given args[0] including increment idea and its current situation, does the current situation need ? seems that it will provide with more context.